scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -111,17 +117,40 @@
         $el.addEventListener('click', function () {
           let active_li = document.querySelector("li.is-active");
           if (active_li) active_li.classList.remove("is-active");
+          prevItem.classList.remove("is-hidden");
+          nextItem.classList.remove("is-hidden");
           $el.parentElement.classList.add("is-active");
-
+          let count = 0;
           let type = $el.getAttribute("data-type");
           let detected_data = detected_values[type];
-          if (detected_data) {
+          if (detected_data.length) {
             setDetectedValues(detected_data);
+            editor.renderer.scrollToLine(detected_data[0].start_line - 1);
           }
           else {
             removeAllMarkers();
+            nextItem.classList.add("is-hidden");
+            prevItem.classList.add("is-hidden");
             editor.getSession().setAnnotations([]);
           }
+
+          prevItem.addEventListener('click', function () {
+            count -= 1
+            if (count < 0){
+              count = 0
```

Comment:
Prev or Next buttons should be disabled when reaching the begin or the end fo the items, ie when there's no more entry to go to.
For example, previous should be disabled by default when loading the detected_data since there is no more previous item at that point.

Reply:
I noticed it working the last time, will fix it ASAP.

Reply:
fixed it at https://github.com/nexB/scancode.io/pull/469/commits/cd674fbec747d342fc7c555e8372deff1489d444, I think its working now as expected.